js/index.js

```diff
@@ -15,9 +24,17 @@ function calculateAll() {
 
   // ITERATION 2
   //... your code goes here
+  const cartItemElements = document.getElementsByClassName('cart-item');
+
+  let totalValue = 0;
+
+  for (cartItemElement of cartItemElements) {
+    totalValue += updateSubtotal(cartItemElement);
+  }
 
   // ITERATION 3
   //... your code goes here
+  const totalValueSpan = document.querySelector('#total-value span').innerHTML = totalValue;
```

Comment:
Since you're not looking to use the stored value then you don't need to store it in a variable. You can just write document.querySelector('#total-value span').innerHTML = totalValue;